src/main/java/com/algorand/algosdk/util/AlgoConverter.java

```diff
@@ -0,0 +1,31 @@
+package com.algorand.algosdk.util;
+
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+public class AlgoConverter {
+
+    private static BigDecimal ALGOS_TO_MICROALGOS_RATIO = BigDecimal.valueOf(1000000L);
```

Comment:
if there is no centralized configs params class in the package, consider making it `final` as well. If there is one, move it there.